docs/core/magnetic_field.md

```
@@ -235,6 +235,23 @@ analytical implementation and is much faster to lookup:
 
+### Multi-range constant field
+
+The multi-range constant field allows to model cases where a magnetic field can
+be described as multiple (potentially overlapping) regions, each of which has
+its own constant magnetic field. This provides more flexibility than the
+{class}`Acts::ConstantBField` while providing higher performance than
+{class}`Acts::InterpolatedBFieldMap`.
+
+This magnetic field provider is configured using a list of pairs, where each
+pair defines a region in three-dimensional space as well as a field vector.
+Magnetic field lookup then proceeds by finding the _last_ region in the
```

Markdownlint (0.35.0)

Expected: asterisk; Actual: underscore
Emphasis style

(MD049, emphasis-style)